pkg/cmd/server/kubernetes/node_config.go

```diff
@@ -317,8 +317,14 @@ func BuildKubernetesNodeConfig(options configapi.NodeConfig, enableProxy, enable
 		if len(options.DNSIP) > 0 {
```

Comment:
Before, you had to pass --enable=dns to get DNS.  Now you don't.  I think we can remove this check.